zulipterminal/ui_tools/messages.py

```diff
@@ -730,11 +753,13 @@ def main_view(self) -> List[Any]:
             )
 
         # Transform raw message content into markup (As needed by urwid.Text)
-        content, self.message_links, self.time_mentions = self.transform_content(
-            self.message["content"], self.model.server_url
-        )
+        (
+            content,
+            self.message_links,
+            self.code_snippets,
+            self.time_mentions,
+        ) = self.transform_content(self.message["content"], self.model.server_url)
```

Comment:
I'm yet to work on adding these tests and was wondering if it would be alright moving this to a separate PR? I'll also include tests for the others as well.

Reply:
Added a test for `self.code_snippets`. I'll add tests for the others in a separate PR 👍

Reply:
Made a PR for tests for the others :)